azurelinuxagent/common/logcollector.py

```diff
@@ -118,7 +119,7 @@ def _set_resource_usage_cgroups(cpu_cgroup_path, memory_cgroup_path):
     @staticmethod
     def _initialize_telemetry():
         protocol = get_protocol_util().get_protocol()
```

Comment:
Looks like get_protocal() still initializing goal state with all properties?

Reply:
Added flag to get_protocol()